WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorViewModel.swift

```diff
@@ -426,6 +426,18 @@ extension ProductSelectorViewModel: SyncingCoordinatorDelegate {
         let lastIndex = productsResultsController.numberOfObjects - 1
         syncingCoordinator.ensureNextPageIsSynchronized(lastVisibleIndex: lastIndex)
     }
+
+    /// Updates the selected filters for the product list
+    ///
+    func updateFilters(_ filters: FilterProductListViewModel.Criteria) {
+        filtersSubject.send(filters)
+    }
+
+    /// Retrieves the latest selected filters from the product list
+    ///
+    func retrieveUpdatedFilters() -> FilterProductListViewModel.Filters {
```

Comment:
💭 this seems to be only called in unit tests and not in the app. generally, we want to avoid making functions just for testing. Instead, we want to test the public states that are affected by `updateFilters` in this case, which is probably the related parameters in `ProductAction.synchronizeProducts` and `ProductAction.searchProducts`.

This might be a bigger change and there are some pre-existing test cases accessing the filters directly. For now, maybe we can update the test cases to access filters through another public variable `ProductSelectorViewModel.filterListViewModel.criteria` instead of this function.